app/src/sharedTest/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentTest.kt

```diff
@@ -418,6 +423,172 @@ class PlatformParametersFragmentTest {
     }
   }
 
+  @Test
+  fun testPlatformParametersFragment_withOnlyOverriddenValue_returnsOverriddenBooleanValue() {
```

Comment:
Here and elsewhere -- Can we explicitly state against which parameter we are testing against?


```suggestion
  fun testPlatformParametersFragment_boolParam_onlyOverriddenValue_returnsOverriddenBoolValue() {
```

Reply:
updated